includes/analytics.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace PFMC\Analytics;
+
+add_action( 'wp_head', __NAMESPACE__ . '\add_ga_tracker' );
+
+/**
+ * Add a Google Analytics tracker to front-end page views.
+ */
+function add_ga_tracker() {
+
+	// Don't track authenticated users.
+	if ( is_user_logged_in() ) {
+		return;
+	}
+
+	?>
+	<!-- Global site tag (gtag.js) - Google Analytics -->
+	<script async src="https://www.googletagmanager.com/gtag/js?id=UA-25661802-1"></script>
```

Comment:
```suggestion
	<script async src="https://www.googletagmanager.com/gtag/js?id=UA-69021927-1"></script>
```
I'm actually not certain about this, but I would assume the GA Measurement ID up here should be theirs, too.

Reply:
🤦‍♂ hahahahahaha good catch!